Assets/MixedRealityToolkit.Examples/Demos/HandTracking/Scenes/NearMenuExamples.unity

```diff
@@ -594,6 +594,11 @@ PrefabInstance:
       propertyPath: autoFollowAtDistance
       value: 0
       objectReference: {fileID: 0}
+    - target: {fileID: 4669628316120752027, guid: d82e2f09994ad2f4fb43359bfb977f64,
```

Comment:
I should see if I can avoid these changes by just modifying the toggle features panel prefab